Assets/Shopify/UIToolkit/Components/RemoteImageLoader.cs

```diff
@@ -55,7 +55,7 @@ public class RemoteImageLoader : MonoBehaviour {
                 if (texture == null) {
                     Debug.LogWarning("Failed to generate texture from image located at " + imageURL);
                     if (failure != null) {
-                        failure(error);
+                        failure("Texture not found");
```

Comment:
Error is null here since we don't actually get an error for this so use our own string.